### PR TITLE
SOC-1547 | There is no way to see other discussions from post details page on mobile

### PR DESCRIPTION
**Pull request by rafalkalinski**
## Links

https://wikia-inc.atlassian.net/browse/SOC-1547

## Description

Implement theme bar above main post that has an arrow pointing left with the text "All Discussions"
arrow and text will be clickable; rest of bar will not
scrolling behavior will remain the same: as a user scrolls down the theme bar will move up and 10px of the bottom will show. When a user scrolls up the theme bar will reappear in full view.

## Reviewers

@bkoval, @kvas-damian

### CODE REVIEW

File: front/main/app/styles/module/_discussion.scss [hunk 1]
```diff
@@ -639,6 +639,9 @@ body.discussions {
 	}
 
 	.post-details-view {
+		.discussion-page {
+			margin-top: 91px;
+		}
```
Comment (bkoval) [inline front/main/app/styles/module/_discussion.scss]: I think we need a blank line after each selector

**Comment by kvas-damian** [comment]
LGTM

**Comment by bkoval** [comment]
LGTM 📃 👯 👪 👡 🎅 💅 👆

File: front/main/app/styles/module/_discussion.scss [hunk 2]
```diff
@@ -639,6 +644,10 @@ body.discussions {
 	}
 
 	.post-details-view {
+		.discussion-page {
+			margin-top: 91px;
```
Comment (hakubo) [inline front/main/app/styles/module/_discussion.scss]: 91?

**Pull request merged**